domains/nLog0312.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Profile card",
+  "repo": "https://github.com/nLog0312/profile-card",
+  "owner": {
+    "username": "nLog0312",
+    "email": "nlog.021203@gmail.com"
+  },
+  "record": {
+    "CNAME": "nlog0312.github.io/profile-card"
```

CNAME can't contain `/`
```suggestion
    "CNAME": "nlog0312.github.io"
```

---

Sorry again, I will fix it right away